Update change-address-contact-and-more.md (#8976)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/manage/change-address-contact-and-more.md

2. On the **Organization profile** tab, select **Organization information**.
3. Update your organization's information, then select **Save changes**. Be sure to fill in all required fields marked with an * to enable saving your changes.

> [!NOTE]

> > SharePoint Online and OneDrive have a 256-character limit on Windows PCs. When you synchronize the document libraries to your PC you will encounter error messages when attempting to do anything within the synchronized document libraries (such as creating folders or renaming documents).

Reviewer: Please replace paragraph with:

SharePoint Online and OneDrive have a 256-character limit on Windows PCs. When you synchronize document libraries to your PC, you will receive error messages when you attempt to do anything within the synchronized document libraries, such as creating folders or renaming documents.

> > Sharepoint online and Onedrive has a 256 character limit on your Windows PC. As users synchronize the document libraries to their PCs, they will encounter the error messages when trying to do anything within synchronized document libraries (i.e., create folders, rename documents, etc.).

Reviewer 2: ```suggestion
> SharePoint Online and OneDrive have a 256-character limit on Windows PCs. When you synchronize the document libraries to your PC you will encounter error messages when attempting to do anything within the synchronized document libraries (such as creating folders or renaming documents).
```
